frontend/src/components/user-profile/user-profile.tsx

```diff
@@ -35,7 +35,7 @@ const UserPage: React.FC = () => {
   const hasUser = Boolean(user);
   const hasPermitToEdit = currentUser?.id === Number(id);
 
-  if (!hasUser) {
+  if (!hasUser && dataStatus === DataStatus.FULFILLED) {
```

Comment:
Maybe it will be better if we put out the expression. For example `const isDataLoaded = dataStatus === DataStatus.FULFILLED`

Reply:
And then `if(!hasUser && isDataLoaded) { }`

Reply:
Agreed, good point 👍
I would be better, but it's okey for now.